src/st/st0/dialogue2.h

```diff
@@ -12,9 +38,17 @@ typedef struct {
     /* 0x10 */ u16 portraitAnimTimer; // portrait animation timer
     /* 0x12 */ u8 unk12;              // unknown
     /* 0x13 */ u8 clutIndex;
-    /* 0x14 */ Primitive* prim[1]; // for dialogue graphics rendering
-    /* 0x18 */ s32 primIndex[8];   // primIndices: unk, actorName, unk
```

Comment:
Didn't see much evidence these were arrays (if they are the other indexes are not used). I've updated this to be closer to the PSP struct which just has a single entry here and padded out the remainder.